x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/hooks/use_index_data.ts

```diff
@@ -49,6 +50,39 @@ function getRuntimeFieldColumns(runtimeMappings: RuntimeMappings) {
   });
 }
 
+function getInitialColumns(indexPattern: any) {
```

Comment:
Could be `IndexPattern` instead of `any`.

Reply:
Good catch - updated in fa3e81bdcbfd5b0fe270287619220c7907a36a59